spec/base_spec.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../base'
+require_relative '../person'
+require 'yaml'
+
+describe Base do
+  before :each do
+    @base = Base.new('Nedjwa')
+  end
+  it 'takes one parameter and returns a base object' do
+    @base.should be_an_instance_of Base
+  end
+  it 'returns name is nedjwa' do
+    @base.nameable.should eql 'Nedjwa'
+  end
+end
+
+describe CapitalizeDecorator do
+  before :each do
+    @capitalize = CapitalizeDecorator.new('Nedjwa')
```

Comment:
- [ ] Kindly note that this method capitalizes the first letter of a word. If I change the name to `nedjwa`, this test should still pass which is not the case right now. Kindly fix this issue.
- **Hint** I think there's a difference between `upcase()` and `capitalize()` methods in Ruby